### DISCUSSION TITLE
Request - Add Librespeed LXC

### DISCUSSION THREAD

**Discussion by soapster** [tteck/Proxmox#2664] [Ideas]
https://github.com/librespeed/speedtest 

This sure would be handy

**Comment by kmmarin** [comment]
I would like this too

**Comment by sharkpunch5** [comment]
me also!

**Reply by tteck** [reply]
https://tteck.github.io/Proxmox/#myspeed-lxc

**Reply by soapster** [reply]
This checks the internet speeds. Libre Speed allows you to run a speedtest from any device (in my case, internal devices - Not exposing it to the internet) to the server.  So you can use it to troubleshoot your home network.
